### DISCUSSION TITLE
Tabele border-collapse: collapse; with dompdf

### DISCUSSION THREAD

**Discussion by Cristicosta** [dompdf/dompdf#2556] [Q&A]
Hello,

I am beginner and I am having troubles using the border-collapse: collapse;.
Everytime that I use this definition on CSS. It picks the tabelle and put it on the other side of the page.

You don t have an exemple that works with border collapse with dompdf that I could learn.

![image](https://user-images.githubusercontent.com/44175251/133791932-3201e85c-ec17-4178-96d3-f10572afcb0d.png)

Happens that the border goes to the other side 

But when I take the border-collapse: collapse; is correct but the tabelle have that lines separating the cells. 
But when I see on the HTML is correct. 

Thank you
Cristina Costa

**Comment by bsweeney** [comment]
Can you share the actual HTML + CSS instead of a screen shot, plus the rendered PDF? It's likely that some other styling is causing your issue.
